src/corelibs/U2Core/src/datatype/U2FeatureType.cpp

```diff
@@ -55,379 +55,209 @@ U2FeatureType U2FeatureTypes::getTypeByName(const QString &visualName) {
 }
 
 QList<U2FeatureTypes::U2FeatureTypeInfo> U2FeatureTypes::initFeatureTypes() {
-    QList<U2FeatureTypeInfo> featureTypeInfos;
```

Comment:
The whole changes in this file are so meaningless and so bloated, so I see no sense of doing that

Reply:
Could you please elaborate on this?
The change reduces number of lines by 2, groups standard annotations and adds links to the standard.